### DISCUSSION TITLE
Munit Test Cases

### DISCUSSION THREAD

**Discussion by kgaurav-mule** [mulesoft-catalyst/metrics-toolkit#71] [Ideas]
Hi,

Thank you for providing such a great framework. I have used it at couple of customer places.
I would like to suggest that we should add some munit test cases  (for the core implementation, not for the dashboards)
I have written few test cases and would like to contribute.Happy to write more test cases if required.

**Comment by olamiral-mulesoft** [comment]
Hey Kumar! 

Thank you for you feedback! Feel free to create a fork and create a pull request containing the MUnit tests you've already implemented. In order to submit a PR, create a new branch starting from develop, push the branch to your forker repo, and create a pull request (always use branch develop on the upstream repo). For more info, please check https://github.com/mulesoft-catalyst/metrics-accelerator/wiki/Team-&-Collaboration
